backend/models/guests.py

```diff
@@ -1,13 +1,12 @@
-from sqlalchemy import Column, Integer, String, Boolean, ForeignKey
-from sqlalchemy.orm import relationship
+from sqlalchemy import Column, ForeignKey, Integer, String, Boolean
 from db import Base
 from enum import Enum as PyEnum
 
 
 class Status(PyEnum):
     pending = "pending"
     attending = "attending"
-    decline = "decline"
+    declined = "decline"
```

Comment:
should the string also be "declined" here?

Reply:
done